src/support/generation.c

```diff
@@ -104,15 +104,18 @@ __wt_gen_next_drain(WT_SESSION_IMPL *session, int which)
 void
 __wt_gen_drain(WT_SESSION_IMPL *session, int which, uint64_t generation)
 {
+    struct timespec start, stop;
     WT_CONNECTION_IMPL *conn;
     WT_SESSION_IMPL *s;
-    uint64_t v, start, stop;
+    uint64_t time_diff, v;
     uint32_t i, session_cnt;
     u_int minutes;
     int pause_cnt;
+    bool verbose_timeout_flags;
 
     conn = S2C(session);
-    start = 0; /* [-Wconditional-uninitialized] */
+    verbose_timeout_flags = false;
+    __wt_epoch(NULL, &start);
```

Comment:
I am wondering why we set start here, as it gets overwritten at line 165, also why the session is NULL with this call.

Reply:
We need to init `start` before it is used. If we don't have this line, we have the warning that we might use it before it is initialized.
I pass in `NULL` as we skip a few instructions when `session` is passed in as `NULL` in `__wt_epoch`.